### PR TITLE
Improve suggestion box

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-react/src/ui/WalletSelector/Suggestions/MultipleSuggestion.tsx [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+import {EnsName} from '../../commons/EnsName';
+import {Spinner} from '../../commons/Spinner';
 
-export const MultipleSuggestion = ({onClick, operationType, suggestion}: SuggestionProps) => (
-  <button className="suggestions-item-btn" id={getSuggestionId(operationType)} onClick={() => onClick(suggestion)}>
-    <p className="suggestions-item-text">{suggestion}</p>
-    <p className="suggestions-item-btn-text">{operationType}</p>
+export const MultipleSuggestion = ({onClick, operationType, suggestion, selectedSuggestion}: SuggestionProps) => (
+  <button disabled={!!selectedSuggestion} className="suggestions-item-btn" id={getSuggestionId(operationType)} onClick={() => onClick(suggestion)}>
+    <div className="suggestions-item-text"><EnsName value={suggestion} /></div>
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/WalletSelector/Suggestions/MultipleSuggestion.tsx]:
```ts
<div className="suggestions-item-text">
    <EnsName value={suggestion} />
</div>
{suggestion !== selectedSuggestion 
    ? <p className="suggestions-item-btn-text">{operationType}</p>
    : <Spinner className='spinner-small-center'/>
}
```

File: universal-login-react/src/ui/WalletSelector/Suggestions/SingleSuggestion.tsx [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
-    <p className="suggestions-hint">{hint}</p>
-    <button className="single-suggestion-btn" id={getSuggestionId(operationType)} onClick={() => onClick(suggestion)}>
-      {operationType}
-    </button>
-  </>
-);
+export const SingleSuggestion = ({onClick, operationType, suggestion, hint, selectedSuggestion}: SingleSuggestionProps) => {
+  return (
+    <>
+      <div className="suggestions-ens-name"><EnsName value={suggestion} /></div>
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/WalletSelector/Suggestions/SingleSuggestion.tsx]: Same formatting

File: universal-login-react/src/ui/commons/EnsName.tsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+interface EnsNameProps {
+  value: string;
+}
+
+export const EnsName = ({value}: EnsNameProps) => {
+  const split = value.split('.');
```
Comment (Marik-D) [inline universal-login-react/src/ui/commons/EnsName.tsx]:
```
const [name, ...rest] = values.split('.')
```

File: universal-login-react/src/ui/commons/EnsName.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+interface EnsNameProps {
+  value: string;
+}
+
+export const EnsName = ({value}: EnsNameProps) => {
+  const [name, ...domain] = value.split('.');
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/commons/EnsName.tsx]: use `parseDomain`

Comment (xitronix) [reply universal-login-react/src/ui/commons/EnsName.tsx]: parseDomain returns just domain. We also need a name here

Comment (xitronix) [reply universal-login-react/src/ui/commons/EnsName.tsx]: @JustynaBroniszewska  Done

File: universal-login-react/src/ui/commons/EnsName.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {parseDomain} from '@universal-login/commons';
+
+interface EnsNameProps {
+  value: string;
+}
+
+export const EnsName = ({value}: EnsNameProps) => {
+  const [name, domain] = parseDomain(value);
+  return <p><b>{name}.</b>{domain}</p>;
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/commons/EnsName.tsx]:
```suggestion
  return <p><b>{name}</b>.{domain}</p>;
```

**Comment by xitronix** [comment]
Screenshot 2019-11-14 at 16 17 18
Screenshot 2019-11-14 at 16 17 32
Screenshot 2019-11-14 at 16 17 26

**Pull request merged**